src/components/node-list/node-list-row.js

```diff
@@ -116,42 +117,49 @@ const NodeListRow = memo(
             dangerouslySetInnerHTML={{ __html: label }}
           />
         </TextButton>
-        {children}
         {type !== 'modularPipeline' && (
-          <label
-            htmlFor={id}
-            className={classnames('pipeline-row__toggle', {
-              'pipeline-row__toggle--disabled': disabled,
-              'pipeline-row__toggle--selected': selected,
-              'pipeline-row__toggle--not-tag': type !== 'tag',
-            })}>
-            <input
-              id={id}
-              className="pipeline-nodelist__row__checkbox"
-              type="checkbox"
-              checked={checked}
-              disabled={disabled}
-              name={name}
-              onChange={onChange}
-            />
-            <VisibilityIcon
-              aria-label={name}
-              className={classnames(
-                'pipeline-nodelist__row__icon',
-                'pipeline-row__toggle-icon',
-                `pipeline-row__toggle-icon--kind-${kind}`,
-                {
-                  'pipeline-row__toggle-icon--parent': Boolean(children),
-                  'pipeline-row__toggle-icon--child': !children,
-                  'pipeline-row__toggle-icon--checked': checked,
-                  'pipeline-row__toggle-icon--unchecked': !checked,
-                  'pipeline-row__toggle-icon--unset': unset,
-                  'pipeline-row__toggle-icon--all-unset': allUnset,
-                }
-              )}
-            />
-          </label>
+          <>
+            {typeof count === 'number' ? (
+              <span
+                onClick={onClick}
+                className={'pipeline-nodelist__row__count'}>
```

Comment:
I don't see any tests relating to the display of the count / existance of this css class for filter type rows - please add a test for this.